packages/tkg-clusterclass-aws/bundle/config/upstream/base.yaml

```diff
@@ -290,6 +290,23 @@ spec:
       openAPIV3Schema:
         type: object
         properties: {}
+  - name: controlPlaneCertificateRotation
+    required: false
+    schema:
+      openAPIV3Schema:
+        type: object
+        properties:
+          activate:
+            type: boolean
+            default: true
+          daysBefore:
+            type: integer
+            format: int32
+            minimum: 7
```

Comment:
What is the behavior if users provide a value less than 7 here?

Reply:
> What is the behavior if users provide a value less than 7 here?

The Cluster "test01" is invalid: spec.topology.variables[0].daysBefore: Invalid value: 6: daysBefore in body should be greater than or equal to 7
: exit status 1

It returns errors as above during creating cluster. And it doesn't fail using --dry-run.